_gtfobins/cpulimit.md

```diff
@@ -0,0 +1,11 @@
+---
+functions:
+  execute-interactive:
+    - description: With -f, the commands are executed in the terminal you start them. -l 100 specified that the command is to use a single CPU core
+    - code: cpulimit -l 100 -f /bin/sh
+  execute-non-interactive:
```

Comment:
I'd drop this function as it doesn't really add nothing useful to the `execute-interactive` case.

Reply:
No I mean remove `execute-non-interactive` and just keep `execute-interactive`.